### ISSUE TITLE
Member Visibility isn't working

### ISSUE THREAD

**Issue by mistyn8** [AaronSadlerUK/Our.Umbraco.UmbNav#79]
### Which *exact* Umbraco version are you using? For example: 8.7.0 - don't just write v8 or v9

11.4.1

### Bug summary

add the Member Visibility options and set up two items, one to hide if logged in the other to hide if logged out.

Both items remain visible in the menu

### Steps to reproduce

see above

**Comment by mistyn8** [comment]
After forking down the repo to investigate... the continue to exit the loop for member visibility is missing adding the item to the removeItems array for later removal..

```
                    if (item.HideLoggedIn && isLoggedIn || item.HideLoggedOut && !isLoggedIn)
                    {
                        removeItems.Add(item);
                        continue;
                    }
```

**Comment by mistyn8** [comment]
PR here - https://github.com/AaronSadlerUK/Our.Umbraco.UmbNav/pull/81

**Comment by AaronSadlerUK** [comment]
Thanks for this, I will check it out soon!

**Comment by AaronSadlerUK** [comment]
This is included in V2.0.7 on NuGet
